validateStudentRepo.sh

```diff
@@ -37,16 +39,15 @@ function validateAllExercises {
     cd $studentRepo
 
     echo $SEPARATOR
-    sbt "gotoExerciseNr 0" 2>&1 | grep "Moved to"
```

Comment:
Are there any courses where gotoExerciseNr 0 doesn't work exactly the same as ICselectFirstExercise?